textattack/shared/utils/misc.py

```diff
@@ -57,34 +56,41 @@ def html_table_from_rows(rows, title=None, header=None, style_dict=None):
     return table_html
 
 
+def get_textattack_model_num_labels(model_name, model_path):
+    """Reads `train_args.json` and gets the number of labels for a trained
+    model, if present."""
+    model_cache_path = textattack.shared.utils.download_if_needed(model_path)
+    train_args_path = os.path.join(model_cache_path, "train_args.json")
+    if not os.path.exists(train_args_path):
+        textattack.shared.logger.warn(
+            f"train_args.json not found in model path {model_path}. Defaulting to 2 labels."
+        )
+        return 2
+    else:
+        args = json.loads(open(train_args_path).read())
+        return args["num_labels"]
```

Comment:
Should we include a default to 2 similar to above here if `num_labels` isn't a key?

Reply:
yes, good idea